src/main/java/com/netchill/webservices/api/MovieWs.java

```diff
@@ -31,4 +31,15 @@ public Production getMovieById(@PathParam("id") Long id) {
         return this.movieService.getMovieById(id);
     }
 
+    @GET
+    @Path("/top-rated")
+    public List<Production> getTopRated() {
+        return movieService.getTopRated();
+    }
+
+    @GET
+    @Path("/top-rated")
+    public List<Production> getTopRatedByGenre(@QueryParam("genre") int genre) {
```

Comment:
Pour les query param, il vaut mieux ne pas utiliser les types primitifs car les QueryParam peuvent être null.
Dans ce cas ça va casser de partout !

```suggestion
    public List<Production> getTopRatedByGenre(@QueryParam("genre") Integer genre) {
```